src/Umbraco.Web.UI.Client/src/views/components/umb-mini-search.html

```diff
@@ -1,6 +1,13 @@
 <ng-form class="umb-mini-search" ng-class="{'--has-value': vm.model !== null && vm.model !== ''}" novalidate>
     <i class="icon icon-search" aria-hidden="true"></i>
+
+    <label class="sr-only" for="search-{{vm.randomUniquId}}">
+        <span ng-if="vm.label.length">{{vm.label}}</span>
+        <localize ng-if="!vm.label.length" key="general_search">Search</localize>
```

Comment:
I would handle `vm.label` in controller so we don't need `ng-if` at all since it can be expensive to use.